### PR TITLE
Homescreen clients desktop version

### PR DESCRIPTION
**Pull request by jfidiles**

### CODE REVIEW

File: _sass/main/_footer.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   }
 
   a:hover {
-    color: $color-text-hover;
+    color: $color-brand;
```
Comment (artisologic) [inline _sass/main/_footer.scss]: nice! 👍

File: _sass/main/_home.scss [hunk 2]
```diff
@@ -137,11 +140,19 @@ $line-spacing-large: 40px;
   display: flex;
   flex-wrap: wrap;
   padding: 0;
-}
 
-.Home-separator {
-  width: 90%;
-  height: 7px;
+  :nth-child(n+9) { /* 1 */
```
Comment (artisologic) [inline _sass/main/_home.scss]: can you explain what this is doing?

File: index.html [hunk 3]
```diff
@@ -77,9 +77,21 @@
...
+                </li>
+                <li class="Home-clientLogo">
+                    <img src="assets/images/homescreen/ebs.png" alt="EBS" />
+                </li>
+                <li class="Home-clientLogo">
+                    <img src="assets/images/homescreen/kpmg.png" alt="KPMG" />
+                </li>
             </ul>
 
-            <img class="Home-separator" src="assets/images/homescreen/horizontal-separator.svg" alt="separator" />
```
Comment (artisologic) [inline index.html]: did you delete that file?

File: _sass/main/_separator.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+  &:before, &:after {
+    $size: 7px;
+
+    content: " ";
+    display: inline-block;
+    width: $size;
+    height: $size;
+    background-color: $color-grey-light;
+    transform: translateY(calc(-50% - 0.5px));
```
Comment (artisologic) [inline _sass/main/_separator.scss]: Can you explain the sizing a bit? why `width: 90%`? and also why each pseudo-element is translated by `-50% -0.5px`? That seems pretty hard-wired.

File: _sass/main/_home.scss (hunk 2 above)
Comment (jfidiles) [inline _sass/main/_home.scss]: Sure. I've seen in the design that only 8 client logos are displayed for mobile devices and 12 in the rest. So I am hiding everything after the 8th logo for mobiles and display them for bigger devices

File: index.html (hunk 3 above)
Comment (jfidiles) [inline index.html]: Yes, I have deleted the svg file and replaced it with another class called "_separator". You noticed that the svg file was scaling down for mobiles so I replaced the file with an <hr> tag and add two squares at the beginning and at the end.

**Comment by artisologic** [comment]
@jfidiles 💪

**Pull request merged**